superx/templates/home.html

```diff
@@ -68,16 +68,19 @@ <h1 id="header" class="display-3 text-center">superX</h1>
         <th colspan="5">העגלה שלי</th>
     </tr>
   <tr>
-        <th scope="col" colspan="2">קוד מוצר</th>
+        <th scope="col" colspan="0" hidden>קוד מוצר</th>
```

Comment:
Why keep this in if it is going to be hidden and never revealed or used?

Reply:
true, not relevant and removed